app/src/main/java/com/github/polypoly/app/MapActivity.kt

```diff
@@ -154,6 +161,48 @@ class MapActivity : ComponentActivity() {
         }
     }
 
+    @Composable
+    fun Hud(data: PlayerGlobalData, round: Int) {
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Box(
+                modifier = Modifier
+                    .padding(16.dp)
+                .background(Color.White)
+                .border(1.dp, Color.Black)
+                .align(Alignment.TopStart)
+            ) {
+                Row(modifier = Modifier.padding(8.dp)) {
```

Comment:
Padding of **8.dp** looks redundant. We could create a class filled with UI constants, such as a universal padding :)!

Reply:
I created a `Padding` class in a package called `utils` to store the constants!